row_cache.hh

```diff
@@ -177,8 +177,8 @@ public:
     public:
         virtual ~external_updater_impl() {}
         virtual future<> prepare() { return make_ready_future<>(); }
-        // FIXME: make execute() noexcept, that will require every updater to make execution exception safe,
-        // also change function signature.
+        // Updaters that are exception safe should handle all exceptions in prepare.
+        // Any exceptions from execute() are fatal.
         virtual void execute() = 0;
```

Comment:
isn't it better to keep the reminder to eventually make execute() noexcept? we're heading there.

Reply:
okay, it's a noble goal indeed.